Makefile

```diff
@@ -31,7 +31,7 @@ truffle-test:
 	docker build . -f ./docker/Dockerfile.truffle -t truffle-test
 	docker-compose -f ./tests/truffle/docker-compose.yml up genesis
 	docker-compose -f ./tests/truffle/docker-compose.yml up -d bsc-rpc bsc-validator1
-	sleep 30
```

Comment:
why?

Reply:
In snap sync, PBSS should wait for all state fully synced. Therefore, sleep more time.